docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,68 +1,189 @@
+All files in a codebase MUST be version controlled. | yes |
+All decisions MUST be documented in commit messages. | no | Commit messages often refer to private issues.
+Every commit message MUST link to discussions and issues wherever possible. | partial | Sometimes these are private connections.
+You SHOULD group relevant changes in commits. | yes | Commits are grouped by pull request.
+You SHOULD mark different versions of your codebase, for example using revision tags or textual labels. | yes | Amsterdam tags new releases on Github using Github releases.
+You SHOULD prefer file formats that can easily be version controlled. | yes | Git-friendliness is in the culture.
 
 ## [Require review of contributions](https://standard.publiccode.net/criteria/require-review.html)
 
-* Ensure commits are reviewed including commit message.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All contributions that are accepted or committed to release versions of the codebase MUST be reviewed by another contributor. | yes |
+Reviews MUST include source, policy, tests and documentation. | | Code yes, but tests and docs/policy not always - maybe a PR template would help?
+Reviewers MUST provide feedback on all decisions made and the implementation in the review. | yes | Weekly dev call for discussion.
+Contributions SHOULD conform to the standards, architecture and decisions set out in the codebase in order to pass review. | yes | Architecture docs for both frontend and backend, might be nice to have a checkbox in a PR template.
+Reviews SHOULD include running both the code and the tests of the codebase. | yes |
+Contributions SHOULD be reviewed by someone in a different context than the contributor. | some | VNG pull requests all reviewed by Amsterdam, frontend & backend cross-review rare, Amsterdam pull requests often not cross-reviewed by VNG. PO acceptance testing not yet part of review.
+Version control systems SHOULD not accept non-reviewed contributions in release versions. | yes |
+Reviews SHOULD happen within two business days. | some |
+Reviews MAY be performed by multiple reviewers. | |
 
 ## [Document your objectives](https://standard.publiccode.net/criteria/document-objectives.html)
 
-* Feature roadmap is not published.
-* Improve goals documented in README to include information on scope, purposes, and reasoning.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+The codebase MUST contain documentation of its objectives – like a mission and goal statement – that is understandable by designers and developers so that they can use or contribute to the codebase. |  | See signalen.org and [technical goals](https://github.com/Amsterdam/signals/blob/master/docs/topics/application-design.md).
+The codebase SHOULD contain documentation on its objectives understandable by policy makers and management. | |
+The codebase MAY contain documentation of its objectives for the general public. | |
```

Comment:
It should is my opinion, and the documentation needs an overhaul (great for the holiday season when we are not at full strength and constrained in what we can deploy to production servers).